packages/qwik/public.d.ts

```diff
@@ -1,4 +1,7 @@
 export {
+  _serialize,
+  _deserialize,
+  _getContextElement,
```

Comment:
This is wrong. We can't add this to the public API, because it's internal API. I think the functions are already exported as internal API here 
https://github.com/QwikDev/qwik/blob/acf5375d70471641fc8e7e4f3c80f6ea93194b1b/packages/qwik/src/core/internal.ts#L12
https://github.com/QwikDev/qwik/blob/acf5375d70471641fc8e7e4f3c80f6ea93194b1b/packages/qwik/src/core/internal.ts#L37

So you need to import it from `@qwik/core/internal`